### PR TITLE
Add Support for msys build

### PR DESCRIPTION
**Pull request by welitonfreitas**
pip not found packpage for mingw enviroment

### CODE REVIEW

File: BuildScripts/create_wheels.sh [hunk 1]
```diff
@@ -77,18 +77,22 @@ function main()
 	# create the wheel for python 3.6
 	build_python_wheel 3 6 1 win32 
 	build_python_wheel 3 6 1 win_amd64
+	build_python_whell 3 6 1 mingw
```
Comment (dyer234) [inline BuildScripts/create_wheels.sh]: hi thanks for contributing. There is a small typo here. would you mind correcting the function name to "wheel" from "whell". After that I should have a chance to run it locally and get a package pushed up to pypi for you.

Comment (welitonfreitas) [reply BuildScripts/create_wheels.sh]: i'm sorry! done!

**Comment by dyer234** [comment]
Thanks for the changes. I merged it, I will be home in a few hours and can do a push to pypi for you then.

**Comment by dyer234** [comment]
When i tried to upload to pypi i received this error:

`Uploading pywinsparkle-1.4.0-cp27-cp27m-mingw.whl
HTTPError: 400 Client Error: Binary wheel 'pywinsparkle-1.4.0-cp27-cp27m-mingw.whl' has an unsupported platform tag 'mingw'. for url: https://upload.pypi.org/legacy/
`

I tried doing `mingw, posix, and cygwin`, do you have any suggestions?

**Comment by dyer234** [comment]
Here, Can you try this:

From this link: https://www.python.org/dev/peps/pep-0425/

It suggests that you can run `distutils.util.get_platform()` and that should be the name that pypi will accept. What name is returned by that?

**Comment by welitonfreitas** [comment]
`distutils.util.get_platform()` return `mingw`.
initially I would use `none-any`, but users from other platforms outside windows could install without receiving error message.

**Comment by dyer234** [comment]
ok. I thought I tried that before and bdist wouldnt create the wheel properly because there were binary files included. I can spend some more time on this

**Comment by dyer234** [comment]
Give it a try now when you get the chance. thanks.

(pip install)

**Pull request merged**